### ISSUE TITLE
Undefined symbols for architecture armv7

### ISSUE THREAD

**Issue by GreenRollingHills** [playgameservices/ios-basic-samples#13]
When my Podfile is:

target 'ButtonClicker' do
    pod 'GooglePlayGames’, ‘~> 5.0’
end

target 'ButtonClickerTests' do
    pod 'GooglePlayGames’, ‘~> 5.0’
end

I get:

Using GTMOAuth2 (1.1.4)
Using GTMSessionFetcher (1.1.7)
Using GoogleAppUtilities (1.1.2)
Using GooglePlayGames (5.1.1)
Using GooglePlusOpenSource (1.7.1)
Using GoogleSignIn (4.0.1)
Using GoogleSymbolUtilities (1.1.2)
Using GoogleToolboxForMac (2.1.0)

And I get the following error:

Undefined symbols for architecture armv7:
  "_OBJC_CLASS_$_GSDK_GTMSessionFetcher", referenced from:
      objc-class-ref in gpg(GPGClearcutLogger.o)
  "_OBJC_CLASS_$_GSDK_GTMSessionFetcherService", referenced from:
      objc-class-ref in gpg(GIPNetworkImage.o)
  "_GTMMethodCheckMethodChecker", referenced from:
      +[NSDictionary(GTMNSDictionaryURLArgumentsAdditions) xxGTMMethodCheckMethodNSString32] in libGoogleToolboxForMac.a(GTMNSDictionary+URLArguments.o)
      +[NSDictionary(GTMNSDictionaryURLArgumentsAdditions) xxGTMMethodCheckMethodNSString33] in libGoogleToolboxForMac.a(GTMNSDictionary+URLArguments.o)
ld: symbol(s) not found for architecture armv7
clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by SotoiGhost** [comment]
Those symbols lives within GoogleNetworkingUtilities framework, add that Framework and try again

**Comment by GreenRollingHills** [comment]
Thanks for the input.  I added that GoogleNetworkingUtilities pod and the demo app compiled without errors.  Then I tried the GooglePlayGames pod without the GoogleNetworkingUtilities pod to see if anything has changed.  With the following pod file:

target 'ButtonClicker' do
    pod 'GooglePlayGames'
end

target 'ButtonClickerTests' do
    pod 'GooglePlayGames'
end

post_install do |installer|
  installer.pods_project.targets.each do |target|
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
    end
  end
end

I get:

Using GoogleAppUtilities (1.1.2)
Using GoogleAuthUtilities (1.0.1)
Using GoogleNetworkingUtilities (1.2.2)
Using GooglePlayGames (5.1.2)
Using GooglePlusOpenSource (1.7.1)
Using GoogleSignIn (2.4.0)
Using GoogleSymbolUtilities (1.1.2)
Using GoogleUtilities (1.3.2)

Note that GoogleNetworkingUtilities is now included.  The ButtonClicker demo app now compiles successfully with just the GooglePlayGames pod.
